examples/cryptopp/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.0)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-cryptopp)
+
+hunter_add_package(cryptopp)
+find_package(cryptopp CONFIG REQUIRED)
+
+
+add_executable(test main.cpp)
+
+
+target_link_libraries(test PRIVATE cryptopp-static)
```

Comment:
It does have both variants but have different names (cryptopp-static, cryptopp-shared)